opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -216,6 +216,16 @@ message Histogram {
   AggregationTemporality aggregation_temporality = 2;
 }
 
+// ExponentialHistogram represents the type of a metric that is calculated by aggregating
+// as a ExponentialHistogram of all reported double measurements over a time interval.
+message ExponentialHistogram {
```

Comment:
Should this get added to `Metric.data` oneof above?

Reply:
+1

Reply:
Done.